monkey/infection_monkey/post_breach/actions/hide_files.py

```diff
@@ -0,0 +1,41 @@
+import time
+from common.data.post_breach_consts import POST_BREACH_HIDDEN_FILES
+from infection_monkey.post_breach.pba import PBA
+from infection_monkey.telemetry.post_breach_telem import PostBreachTelem
+from infection_monkey.utils.hidden_files import\
+    get_commands_to_hide_files,\
+    get_commands_to_hide_folders,\
+    cleanup_hidden_files,\
+    get_winAPI_to_hide_files
+from infection_monkey.utils.environment import is_windows_os
+
+
+CREATE_HIDDEN = [get_commands_to_hide_files,
+                 get_commands_to_hide_folders]
+
+
+class HiddenFiles(PBA):
+    """
+    This PBA attempts to create hidden files and folders.
+    """
+
+    def __init__(self):
+        super(HiddenFiles, self).__init__(name=POST_BREACH_HIDDEN_FILES)
+
+    def run(self):
+        # create hidden files and folders
+        for method_to_create in CREATE_HIDDEN:
```

Comment:
`method_to_create` and `CREATE_HIDDEN` aren't the best names, IMO. `CREATE_HIDDEN` has Functions, not Methods. And these functions don't create hidden FSOs, they return the commands that create hidden FSOs.

Maybe refactor this name to something that's more close to the actual meaning. `get_hidden_FSO_creation_commands`? Sounds like a bad name as well, so think about it some more 😅

Reply:
How about 
```python3
for function_to_get_commands in HIDDEN_FSO_CREATION_COMMANDS
```
?

Reply:
Much better 👍👍